docs/source/conf.py

```diff
@@ -283,3 +285,53 @@
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
+
+class HTML5Translator(SphinxHTML5Translator):
+    """
+    This is our custom HTML5Translator which extends the one provided by Sphinx.
+    """
+    STRONG_TO_H3_HEADINGS = [
+        'Example',
+        'Examples',
+        'Exceptions',
+        'Request Syntax',
+        'Response Structure',
+        'Response Syntax',
+        'Structure',
+        'Syntax'
+    ]
+
+    def visit_admonition(self, node, name=''):
+        """Uses the h3 tag for admonition titles instead of the p tag"""
+        self.body.append(self.starttag(
+            node, 'div', CLASS=('admonition ' + name)))
+        if name:
+            title = (
+                f"<h3 class='admonition-title'>"
```

Comment:
```suggestion
                "<h3 class='admonition-title'>"
```

Reply:
Also we're mixing double and single quotes in this file. I know there's some mixing in other places, but can we try to stick to one in this PR? I'd vote for double quotes.